native/cocos/platform/android/java/src/com/cocos/lib/CocosVideoView.java

```diff
@@ -252,6 +254,8 @@ public void stop() {
 
             // after the video is stop, it shall prepare to be playable again
             try {
+                mMediaPlayer.reset(); // reset to avoid some problems on start.
```

Comment:
Why need to load data source again in `stop`? It will take IO resource and cost some time to load it again.

Reply:
Yep, the same question.

Reply:
if not reload data， next start the video will cause a bug, getCurrentTime always return 0. 
refer to https://stackoverflow.com/questions/4486828/android-media-player-restart-audio-after-calling-stop

Reply:
But you call `loadDataSource ` again after `reset`.

Reply:
> But you call `loadDataSource ` again after `reset`.

After [reset()](https://developer.android.com/reference/android/media/MediaPlayer#reset()), the object is like being just created.

Reply:
What error if not `reset()` before reload data?

Reply:
start play , but getCurrentTime always return 0 , this bug: cocos#7845